test/lib/docker.test.ts

```diff
@@ -253,13 +253,77 @@ test("findGlobs", async (t) => {
     t.same(files, ["/pom.xml", "/dir3/file6.csproj"]);
   });
 
-  t.test("find globs using registry globs and fixture", async (t) => {
+  t.test(
+    "find globs on ghost app using node_modules exclude glob",
+    async (t) => {
+      stub.resolves({
+        stdout: readFileSync(getLSOutputFixture("ghost-app.txt")).toString(),
```

Comment:
is `txt` the suitable format here?

Reply:
yes, I would think so, this is the captured output of "ls -1apR /" from the running container.